frontend/src/_components/AppModal.jsx

```diff
@@ -163,7 +161,6 @@ export function AppModal({
               style={{
                 borderColor: errorText ? '#DB4324 !important' : 'initial',
               }}
```

Comment:
Any specific reason for removing `disabled` attribute from this button? If not we need to add it back